build.cmd

```diff
@@ -1,10 +1,5 @@
-copy .\src\Stripe\bin\Release\Stripe.net.dll .\build\net45\Stripe.net.dll /Y
-copy .\src\Stripe\bin\Release\Stripe.net.xml .\build\net45\Stripe.net.xml /Y
-copy .\src\Stripe.Portable\bin\Release\Stripe.net.dll .\build\portable\Stripe.net.dll /Y
-copy .\src\Stripe.Portable\bin\Release\Stripe.net.xml .\build\portable\Stripe.net.xml /Y
+dotnet pack -c Release src\Stripe.net
 
-cd build
-del *.nupkg
-nuget pack Stripe.net.nuspec
+copy .\src\Stripe.net\bin\Release\Stripe.net.*.nupkg .\build\
 
```

Comment:
The namespace is still the same, I just needed to rename the folder, so `dotnet pack` creates the correct NuGet ID, since you can't specify that in `project.json`